hifi_vcpkg.py

```diff
@@ -195,7 +195,7 @@ def bootstrap(self):
             downloadVcpkg = True
 
         if downloadVcpkg:
-            if "HIFI_VCPKG_BOOTSTRAP" in os.environ:
+            if "HIFI_VCPKG_BOOTSTRAP" in os.environ and 'aarch64' != platform.machine():
```

Comment:
This should emit some sort of prominent message, because otherwise we have a setting that the user is intentionally try to apply, but is being completely and silently ignored.

Reply:
I initially wanted to do this in the github workflow files, but couldn't find an easy way to set environment variables conditionally, and was in a scramble to get the builds working after windows environment updated. The bootstrapping worked for a brief period until something updated again, so I ended up downgrading the environment and am now wondering if I should just revert the changes that were meant to fix it, and keep them in mind for whenever we get around to updating our vcpkg setup in general. 

On the other hand might try fixing the boostrapping on aarch64 linux, apparently there is an extra parameter that needs to be specified: https://github.com/microsoft/vcpkg/discussions/17738